x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/FlushJobAction.java

```diff
@@ -187,6 +212,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             if (skipTime != null) {
                 builder.field(SKIP_TIME.getPreferredName(), skipTime);
             }
+            builder.field(REFRESH_REQUIRED.getPreferredName(), refreshRequired);
```

Comment:
Similarly, it shouldn't be included in the JSON representation. (Like `waitForNormalization` is not mentioned in this method.)